### PR TITLE
Added Joel's Advice

### PR DESCRIPTION
**Pull request by joelrottenberg**
I add on Joel's advice for new programmers.

### CODE REVIEW

**Comment by acodeinprogress** [comment]
Hey @joelrottenberg,

This looks great! I just have 1 teensy request before I merge this in.
We would like for you to include a link to your advice page in the index.html file. If you could add that in, all you have to do is add, commit, and push your changes and they will join your pull request here. Once you do that, we can get it all merged in.

xoxo,
Sharon

**Comment by joelrottenberg** [comment]
It took me a while to understand what this message was about.

I actually put a link on top of the page, I'm not sure what you mean?

Joel

**Comment by acodeinprogress** [comment]
Hi @joelrottenberg,

We'll need for you to edit the `index.html` file in your text editor and add a link that will take the user to your advice page.

xoxo,
Sharon

File: index.html [hunk 1]
```diff
@@ -3,16 +3,18 @@
 
 <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="css/joel-style.css">
```
Comment (redrambles) [inline index.html]: Hey @joelrottenberg - we don't want to add a link to your personal stylesheet here in the main index.html page - just the link to your index page, which you did on line 17 (great!). Please remove line 6, add and commit the change and push your branch back up. :)

**Comment by redrambles** [comment]
Hey @joelrottenberg,

Thanks for making that change! Your code is all merged in! Yay!

Cheers,
Ann :)

**Comment by joelrottenberg** [comment]
Great.

**Pull request merged**